markdown/3.x/en/ff-asn.md

```diff
@@ -147,9 +147,11 @@ Slider groups are handled in a different way. To style them you need to use the
 <ff-asn-group-slider></ff-asn-group-slider>
 ```
 
-A slider can have a `[slot="groupCaption"]` attribute but instead of detailedLinks and hiddenLinks the slider group accepts an `ff-slider-control` and/or an `ff-slider` element.
+A slider can have a `[slot="groupCaption"]` attribute and a `[data-container="removeFilter"]` element but instead of `detailedLinks` and `hiddenLinks` the slider group accepts an `ff-slider-control`.
```

Comment:
Is it mandatory to have both `[slot="groupCaption]` and `[data-container="removeFilter"]` in the same time to replace `detailedLinks` and `hiddenLinks` with `ff-slider-control`? I think they're both optional, but when reading this sentence I understood this like: 'you can replace 'detailedLinks' and 'hiddenLinks' with 'ff-slider-control' when both `[slot="groupCaption"]` attribute and `[data-container="removeFilter"]` are present'. Please, try to express this in a way that does not allow similar interpretations.

Reply:
As it is already stated above, that the slider works differently, I think mentioning elements which are not supported is only confusing. What do you think about [my suggestion below](https://github.com/FACT-Finder-Web-Components/documentation/pull/195/files#r294187097)?